map/query-features-with-arcade-expression/src/main/java/com/esri/samples/query_features_with_arcade_expression/QueryFeaturesWithArcadeExpressionSample.java

```diff
@@ -0,0 +1,197 @@
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.ArcGISFeature;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+
+public class QueryFeaturesWithArcadeExpressionSample extends Application {
+
+  private MapView mapView;
+  private ArcGISMap map;
+  private Layer policeBeatsLayer;
+  private final String rpdBeatsLayerName = "RPD Beats  - City_Beats_Border_1128-4500";
+  private final String calloutText = "Crimes in the last 60 days: ";
+
+  @Override
+  public void start(Stage stage) {
+    try {
+      // create stack pane and application scene
+      var stackPane = new StackPane();
+      var scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Query Features With Arcade Expression Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a progress indicator
+      var progressIndicator =  new ProgressIndicator();
+
+      // create a portal item and use it to create a map
+      var portal = new Portal("https://www.arcgis.com/");
+      var portalItem = new PortalItem(portal, "539d93de54c7422f88f69bfac2aebf7d");
+      map = new ArcGISMap(portalItem);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set callout leader to appear at the bottom of the callout
+      mapView.getCallout().setLeaderPosition(Callout.LeaderPosition.BOTTOM);
+
+      // wait for the map to finish loading data from portal before accessing its layers
+      map.addDoneLoadingListener(() -> {
+        if (map.getLoadStatus() == LoadStatus.LOADED) {
+          for (Layer operationalLayer : map.getOperationalLayers()) {
+            // if the operational layer is the RPD Beats layer, show it on the map and hide the progress indicator
+            if (operationalLayer.getName().equals(rpdBeatsLayerName)) {
+              policeBeatsLayer = operationalLayer;
+              operationalLayer.setVisible(true);
+              progressIndicator.setVisible(false);
+            } else {
+              // if the operational layer is any layer other than the RPD Beats layer, don't show it.
+              operationalLayer.setVisible(false);
+            }
+          }
+          // if map clicked, evaluate an arcade expression at that point
```

Comment:
new line here for readability
```suggestion
          }
          
          // if map clicked, evaluate an arcade expression at that point
```